DI-2086/extract_vaf_dp_filter.py

```
@@ -0,0 +1,201 @@
+    ------
+    FileNotFoundError
+        If vcf_path is None, empty or not found
+    """
+
+    # Raise an error if file is not found
+    if not vcf_path:
+        raise FileNotFoundError(f"VCF file not identified in {vcf_path}")
+
+    # Run the bcftools command
+    try:
+        result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)